app/classes/Transvision/ShowResults.php

```diff
@@ -397,6 +412,10 @@ public static function resultsTable($search_object, $search_results, $page)
             $anchor_name = str_replace(['/', ':'], '_', $key);
 
             $clipboard_target_string  = 'clip_' . md5($target_string);
+            if ($locale2 == 'sr') {
+                $transliterated_string = self::getTransliteratedString($target_string, 'sr-Cyrl');
+                $transliterate_id = substr($clipboard_target_string, 5);
```

Comment:
I don’t think we should reuse the clipboard id, maybe something like that?
`$transliterate_id = 'transliterate_' . md5($target_string);`